Testscripts/Linux/verify_vmbus_interrupts.sh

```diff
@@ -59,12 +62,15 @@ function verify_vmbus_interrupts() {
         fi
     done < "/proc/interrupts"
 
-    if [ $nonCPU0inter -ne "$cpu_count" ]; then
-        LogErr "Not all CPU cores are processing VMBUS interrupts!"
+    if [[ $nonCPU0inter -eq 0 ]]; then
+        LogErr "No CPU core is processing VMBUS interrupts!"
         SetTestStateFailed
         exit 0
     fi
 
+    if [[ $cpu_count -gt 4 && $nonCPU0inter -lt "$cpu_count" ]]; then
```

Comment:
I think we can use ```if [ $nonCPU0inter -ne "$cpu_count" ]; then``` to notify the missing interrupt

Reply:
I don't get it why we need.